board.py

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
this class looks good. 
where is this file in the directory structure?
it should be in CS5551/mysite/checkers

Reply:
I don’t know about the file directory structure. Simply i put it in my branch that is usman_work.